belgif-rest-problem/src/test/java/io/github/belgif/rest/problem/validation/RequestValidatorTest.java

```diff
@@ -19,302 +17,432 @@
 import io.github.belgif.rest.problem.api.InputValidationIssues;
 
 class RequestValidatorTest {
```

Comment:
There doesn't seem to be a test which chains multiple validations and verifies they are all checked (i.e. validation does not fail fast)?